docs/modules/registry.md

```diff
@@ -39,6 +39,17 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 - `string`, the Docker image to use.
 - `testcontainers.ContainerCustomizer`, a variadic argument for passing options.
 
+### Docker Auth Config
+
+The module exposes a way to set the Docker Auth Config for the Registry container, thanks to the `SetDockerAuthConfig` function.
+This is useful when you need to pull images from a private registry. It basically sets the `DOCKER_AUTH_CONFIG` environment variable
+with authentication for with the given host, username and password sets. It returns a function to reset the environment back to the previous state,
```

Comment:
Looks like a typo here, think the with should be removed

Reply:
Actually copied from the function comments. Fixed in both places